src/org/feedhenry/GitHubUtils.groovy

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/groovy
+package org.feedhenry
+
+import groovy.json.JsonSlurperClassic
+
+def ghApiRequest(String endpoint, String httpMethod = 'GET', String requestBody = null, String credentialsId = 'githubautomatron', customHeaders = []) {
```

Comment:
Not a big deal, but do we know of a suitable return type for this? I just noticed that the method after this has a return type of String, but this is just a def.